apps/backoffice-v2/src/pages/Entity/Entity.page.tsx

```diff
@@ -1,12 +1,10 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
 import { BlocksVariant } from '@/lib/blocks/variants/BlocksVariant/BlocksVariant';
-import { useEntityLogic } from '@/pages/Entity/hooks/useEntityLogic/useEntityLogic';
+import { useCurrentCase } from '@/pages/Entity/hooks/useCurrentCase/useCurrentCase';
 import { Case } from './components/Case/Case';
 
-import { ProcessTracker } from '@/common/components/molecules/ProcessTracker/ProcessTracker';
-import { TWorkflowById } from '@/domains/workflows/fetchers';
-
 export const Entity = () => {
-  const { workflow, selectedEntity, plugins, processTrackerProcesses } = useEntityLogic();
+  const { workflow, selectedEntity } = useCurrentCase();
```

Comment:
Revert the name of the hook, the intention was that this hook is only supposed to be used in this component.
```suggestion
  const { workflow, selectedEntity } = useEntityLogic();
```